package.json

```diff
@@ -12,8 +12,8 @@
     "grunt-eslint": "^13.0.0",
     "grunt-requirejs": "^0.4.2",
     "mobify-code-style": "^2.2.1",
-    "nightwatch": "0.7.9",
-    "nightwatch-commands": "1.1.1"
+    "nightwatch": "0.8.16",
```

Comment:
Derrick mentioned to downgrade to nightwatch to 0.8.15 today, do we need to update here?